src/main/java/lotto/Application.java

```diff
@@ -1,7 +1,35 @@
 package lotto;
 
+import camp.nextstep.edu.missionutils.Console;
+import model.domain.BonusNumber;
+import model.domain.LottoRankResult;
+import model.domain.Lottos;
+import model.domain.PurchaseAmount;
+import model.domain.WinningLotto;
+import model.domain.WinningNumbers;
+import model.service.LottoMakerService;
+import model.service.LottoRankResultMakerService;
+import view.BonusNumberInput;
+import view.InputString;
+import view.OutputView;
+import view.PurchaseAmountInput;
+import view.WinningNumbersInput;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        InputString<PurchaseAmount> purchaseAmountInput = new InputString<>(new PurchaseAmountInput());
+        PurchaseAmount purchaseAmount = purchaseAmountInput.getInputString(Console::readLine);
```

Comment:
현재 Controller가 Console::readLine을 통해 직접 사용자 입력을 받고 있어서 책임 분리가 잘 되지 않은 것 같습니다.

```suggestion
        String input = inputView.inputWinningNumbers();
        WinningNumbers winningNumbers = winningNumbersParser.parse(input);
```

이렇게 나누어서 컨트롤러는 흐름만 제어하는 것이 좋은 방향인 것 같습니다!

Reply:
조언 감사합니다 !!